manifests/b/B3log/SiYuan/2.0.0-beta1/B3log.SiYuan.installer.yaml

```diff
@@ -0,0 +1,44 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: B3log.SiYuan
+PackageVersion: 2.0.0-beta1
```

Comment:
Beta
![ThatsANoGoNicCageGIF](https://user-images.githubusercontent.com/15158490/163988678-c92808f6-f39d-434f-a3c2-9b267fdb87e3.gif)

Reply:
Developer marked it as a non pre-release.